models/certificateTemplates.js

```diff
@@ -1,29 +1,28 @@
 module.exports = {
-  name: "certificateTemplates",
-  schema: {
-    templateUrl: String,
-    issuer: {
-      type : Object,
-      required : true
-    },
-    status: {
-      type : String,
-      required : true,
-      default : "ACTIVE"
-    },
-    solutionId: {
-      type : "ObjectId",
-      index : true,
-      unique : true
-    },
-    programId: {
-      type : "ObjectId",
-      index : true,
-      required : true
-    },
-    criteria: {
-      type : Object,
-      required : true
-    }
-  }
-};
\ No newline at end of file
+	name: 'certificateTemplates',
```

Comment:
@Prajwal17Tunerlabs Do add corresponding migrations to create default entries for various certificates here

Reply:
once i receive svg templates i will add script to create default certificate-templates